pkgs/development/python-modules/keymap-drawer/default.nix

```diff
@@ -18,18 +18,18 @@
   versionCheckHook,
 }:
 let
-  version = "0.22.1";
+  buildPythonPackage' = fn: lib.fix (self: buildPythonPackage (fn self));
```

Comment:
If this transition is that simple, why don't we go all the way and do it in `pkgs/development/interpreters/python/mk-python-derivation.nix`?

Reply:
Currently I'm imagining that `buildPythonPackage` will need to provide access to multiple "final" attrsets:
1. `pythonAttrs` the final attrs passed to `buildPythonPackage` or `overridePythonAttrs`
2. `finalAttrs` the final attrs passed to the underlying `mkDerivation` call or `overrideAttrs`
3. `finalPackage` the final package produced by `mkDerivation` or `overrideAttrs`

Internally, we could get the last two from `mkDerivation`. We'll have to create our own `pythonAttrs` fixpoint, I'm guessing this will need to be done in `makeOverridablePythonPackage` or a wrapper of it.